assets/js/src/ajax.js

```diff
@@ -55,8 +59,8 @@ MailPoet.Ajax = {
     }
 
     // set default timeout
-    if (this.options.token === null) {
-      this.options.timeout = 0;
+    if (this.options.timeout === null) {
+      this.options.timeout = 2000;
```

Comment:
Is this change intentional? Do you really want to limit all requests to 2 seconds?

Reply:
This is definitely my mistake. I tested an error message and I didn't notice this change after that. Thank you